aim/sdk/repo.py

```diff
@@ -81,16 +108,18 @@ def __init__(self, path: str, *, read_only: bool = None, init: bool = False):
                 unmount_remote_repo(self.root_path, self._mount_root)
             raise RuntimeError(f'Cannot find repository \'{self.path}\'. Please init first.')
 
-        self.container_pool: Dict[ContainerConfig, Container] = WeakValueDictionary()
+        self.container_pool: WeakValueDictionary[ContainerConfig, Container] = WeakValueDictionary()
         self.persistent_pool: Dict[ContainerConfig, Container] = dict()
-        self.container_view_pool: Dict[ContainerConfig, Container] = WeakValueDictionary()
+        self.container_view_pool: WeakValueDictionary[ContainerConfig, Container] = WeakValueDictionary()
```

Comment:
I remember having issues with genertic typings of `WeakValueDictionary`.
Please check these with Python3.6.

Reply:
Yup, tests do not pass with Python 3.6. We can either use Dict or a more generic MutableMapping.

Reply:
Tests do not pass, but somewhere else, not here. `WeakKeyDictionary` is the problematic one.

Reply:
Yes, there was an issue when using `WeakKeyDictionary[]` typing with generic `T`. I believe we should avoid using `WeakValueDictionary` and `WeakKeyDictionary` as type annotations (including here).

Reply:
yeah, will change back to `Dict`